pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -18913,5 +18901,4 @@ final: prev:
     meta.homepage = "https://github.com/jhradilek/vim-snippets/";
   };
 
-
```

Comment:
note: this will just produce merge conflicts, and it will get back next time someone run update script

Reply:
Thanks, missed it when I was reverting changes in there.